src/createServer.js

```diff
@@ -1,11 +1,21 @@
 'use strict';
 
-// const express = require('express');
+const express = require('express');
+const { reset: resetUsers } = require('./services/users');
+const { reset: resetExpenses } = require('./services/expenses');
+const { router: userRouter } = require('./routes/users');
+const { router: expensesRouter } = require('./routes/expenses');
```

Comment:
I would call it `expensesRouter` directly in file `./routes/expenses` to avoid renaming. It's true for all imports.